clients/admin-ui/src/features/privacy-experience/JavaScriptTag.tsx

```diff
@@ -11,38 +14,35 @@ import {
   Text,
   useDisclosure,
 } from "fidesui";
-import { useMemo, useRef } from "react";
+import { useMemo, useRef, useState } from "react";
 
 import ClipboardButton from "~/features/common/ClipboardButton";
-import { useFeatures } from "~/features/common/features";
 import { CopyIcon } from "~/features/common/Icon";
-import { useGetFidesCloudConfigQuery } from "~/features/plus/plus.slice";
-
-const PRIVACY_CENTER_HOSTNAME_TEMPLATE = "{privacy-center-hostname-and-path}";
-const FIDES_JS_SCRIPT_TEMPLATE = `<script src="https://${PRIVACY_CENTER_HOSTNAME_TEMPLATE}/fides.js"></script>`;
-const FIDES_GTM_SCRIPT_TAG = "<script>Fides.gtm()</script>";
+import {
+  FIDES_JS_SCRIPT_TEMPLATE,
+  PRIVACY_CENTER_HOSTNAME_TEMPLATE,
+  removePropertyIdFromScript
+} from "./fidesJsScriptTemplate";
```

Comment:
I have no idea when we want to use relative vs absolute imports as a "general rule". Personally I prefer all absolute but I don't know if that's our general practice.